cogstat/cogstat_stat.py

```diff
@@ -584,22 +576,26 @@ def variable_pair_regression_coefficients(slope, intercept, std_err, intercept_s
             regression_coefficients += '\n' + '<decision>' + _('Assumption of homoscedasticity for CI '
                                                                'calculations met.') + '</decision>'
 
+        if len(x) > 1:
+            if multicollinearity is None:
+                regression_coefficients += '\n' + '<decision>' + _('Multicollinearity could not be calculated.') + ' ' +\
+                                                       _('Point estimates and CIs may be inaccurate.')  + '</decision>'
+            elif multicollinearity:
+                regression_coefficients += '\n' + '<decision>' \
+                                           + _('Multicollinearity suspected.') + ' ' + \
+                                           _('Point estimates and CIs may be inaccurate.') + '</decision>'
+            else:
+                regression_coefficients += '\n' + '<decision>' + _('Assumption of multicollinearity for'
+                                                                   ' CI calculations met.') + '</decision>'
 
-        # Calculate 95% CIs for slope and intercept
-        # https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.linregress.html
-        from scipy.stats import t
-        tinv = lambda p, df: abs(t.ppf(p / 2, df))
-        ts = tinv(0.05, n - 2)
-        slope_ci_low = slope - ts * std_err
-        slope_ci_high = slope + ts * std_err
-        intercept_ci_low = intercept - ts * intercept_stderr
-        intercept_ci_high = intercept + ts * intercept_stderr
+        # Gather point estimates and CIs into table
+        cis = results.conf_int(alpha=0.05)
 
-        pdf_result.loc[_("Slope")] = \
-            ['%0.3f' % slope, '[%0.3f, %0.3f]' % (slope_ci_low, slope_ci_high)]
+        for i in x:
+            pdf_result.loc[_("Slope "+i)] = ['%0.3f' % (results.params[i]), '[%0.3f, %0.3f]' % (cis.loc[i,0], cis.loc[i,1])]
```

Comment:
_("Slope "+i) cannot be localized. Did you mean _('Slope %s') %s ?

Use ' instead of "

Maybe I'm lost in the code, but I can't see how this will create appropriate table for multiple regressors. Please, double check if pdf_result looks OK

Reply:
String changed. Function is now appropriate for multiple regressors.